bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WH_WToLNu_1j_SMEFTsim_topU3l/WH_WToLNu_1j_SMEFTsim_topU3l_reweight_card.dat

```diff
@@ -0,0 +1,2761 @@
+# reweight card file created with https://github.com/schoef/gridpacks on 2021-04-18 22:34:04
+# Command line arguments: make_reweight_card.py --couplings 2 cHj1 1 cHj3 1 cHu 1 cHd 1 cHudRe 1 cHudIm 1 cHW 1 cHWtil 1 cHB 1 cHBtil 1 cHWB 1 cHWBtil 1 cHbox 1 cHDD 1 cbHRe 1 cbHIm 1 --filename SMEFTsim_VH_reweight_card.dat --overwrite
+# Polynomial order: 2
+change rwgt_dir rwgt
+
+launch --rwgt_name=dummy # Name of first argument seems to be rwgt_1. Add dummy to fix it.
+
```

Comment:
Please remove. I fixed this bug already a while ago. No need to have a dummy.

Reply:
Done